plugins/modules/backup_vault.py

```diff
@@ -0,0 +1,340 @@
+    choices: ['present', 'absent', 'enabled', 'disabled']
+    default: present
+  backup_vault_name:
+    description:
+      - Name for the Backup Vault.
+      - Names are  unique to the account used to create them and the Amazon Web Services Region where they are created.
+      - They consist of letters, numbers, and hyphens.
+    type: str
+    required: true
+  backup_vault_tags:
+    description:
+      - Metadata that you can assign to help organize the resources that you create.
+      - A dictionary of tags to add or remove from the resource.
+      - If the value provided for a tag key is null and I(state=absent), the tag will be removed regardless of its current value.
+    type: dict
+    aliases: ['tags']
+  encryption_key_arn:
+    description:
+      - The server-side encryption key that is used to protect the backups.
+    type: str
+  creator_request_id:
+    description:
+      - A unique string that identifies the request and allows failed requests to be retried without the risk of running the operation twice.
+      - If used, this parameter must contain 1 to 50 alphanumeric or ‘-_.’ characters.
+    type: str
+notes:
+  - The I(purge_tags) option was added in release 4.0.0
+
+extends_documentation_fragment:
+  - amazon.aws.common.modules
+  - amazon.aws.region.modules
+  - amazon.aws.tags
+  - amazon.aws.boto3
+"""
+
+EXAMPLES = r"""
+- name: create single region cloudtrail
+  amazon.aws.backup_vault:
+    state: present
+    backup_vault_name: default-vault
+    encryption_key_arn: arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab
+    tags:
+      environment: dev
+      Name: default
+
+"""
+
+RETURN = r"""
+exists:
+    description: whether the resource exists
+    returned: always
+    type: bool
+    sample: true
+backup_vault:
+    description: BackupVault resource details
+    returned: always
+    type: complex
+    sample: hash/dictionary of values
+    contains:
+        backup_vault_name:
+            description: The name of a logical container where backups are stored.
+            returned: success
+            type: str
+            sample: default-name
+        backup_vault_arn:
+            description: An Amazon Resource Name (ARN) that uniquely identifies a backup vault.
+            returned: success
+            type: str
+            sample: arn:aws:backup:us-east-1:123456789012:vault:aBackupVault
+        creation_date:
+            description: The date and time a backup vault is created, in Unix format and Coordinated Universal Time (UTC).
+            returned: success
+            type: datetime
+            sample: 1516925490.087 (represents Friday, January 26, 2018 12:11:30.087 AM).
+        tags:
+            description: hash/dictionary of tags applied to this resource
+            returned: success
+            type: dict
+            sample: {'environment': 'dev', 'Name': 'default'}
+"""
+
+try:
+    from botocore.exceptions import ClientError, BotoCoreError
```

Comment:
```suggestion
    from botocore.exceptions import ClientError, BotoCoreError
```

I generally prefer to simply import botocore and then fully qualify references to them this makes it easier if you then need to pull in other exceptions.